Moved the order of tasks

bugweis opened the pull request:
Starting of the NFS service was moved to the end of the role.

Fixes the situation when there are no existing exported directories
before the play is played.

bugweis committed:
Moved the order of tasks

Starting of the NFS service was moved to the end of the role.

Fixes the situation when there are no existing exported directories
before the play is played.

atsaki commented:
The play doesn't work on the situation when there are no existing exported directories.
If so, can you provide me error message?

bugweis commented:
Sorry for the late reply. I agree that the comment is a bit confusing and it took me a while to remember what I meant. Finally I figured out. Consider following situation:

The `/etc/exports` file contains a row that points to a non-existing directory:

```
  /asd  192.168.80.1(ro,sync,no_subtree_check)
```

When you try to start NFS service manually it outputs following:

```
root@devel:/# service nfs-kernel-server start
 * Exporting directories for NFS kernel daemon...
exportfs: Failed to stat /asd: No such file or directory
                                                                [ OK ]
 * Starting NFS kernel daemon                                   [ OK ]
```

When the ansible role is played in the same situation it outputs following:

```
ansible-playbook install/development.yml --inventory=install/development.ini

...

TASK: [nfs-server | Include OS specific variables] **************************** 
ok: [devel]

TASK: [nfs-server | Install nfs-utils when ansible_os_family is RedHat] ******* 
skipping: [devel]

TASK: [nfs-server | Install nfs-kernel-server when ansible_os_family is Debian] *** 
ok: [devel] => (item=nfs-kernel-server)

TASK: [nfs-server | Ensure NFS service started and enabled] ******************* 
changed: [devel] => (item=nfs-kernel-server)

TASK: [nfs-server | Ensure exported directory exists] ************************* 
changed: [devel] => (item={'path': '/asd', 'hosts': [{'name': '192.168.80.1', 'options': ['ro', 'sync', 'no_subtree_check']}]})

TASK: [nfs-server | Ensure exported directories are in /etc/exports] ********** 
ok: [devel] => (item={'path': '/asd', 'hosts': [{'name': '192.168.80.1', 'options': ['ro', 'sync', 'no_subtree_check']}]})
```

The directory is created but the _refresh exports_ handler (`exportfs -ra`) is not run.

I incorrectly assumed that when non-existing directory is being exported it is simply skipped with the error message from above: `exportfs: Failed to stat /asd: No such file or directory` but according to my tests it is exported anyway because if I create the directory _after_ it has been exported (the NFS service is started) then the directory is actually accessible from the client machine.

To sum up: according to my latest test all this commit fixes is that NFS service doesn't produce the error. It just seems a bit more logical to me that the service is started _after_ the directory is eventually created.
